lib/instances/LocalizationTable.lua

```diff
@@ -0,0 +1,34 @@
+local BaseInstance = import("./BaseInstance")
+local HttpService = import("./HttpService")
+local InstanceProperty = import("../InstanceProperty")
+local LocalizationTable = BaseInstance:extend("LocalizationTable")
+
+local httpService = HttpService:new()
+
+LocalizationTable.properties.SourceLocaleId = InstanceProperty.normal({
+	getDefault = function()
+		return "en-us"
+	end,
+})
+
+LocalizationTable.properties.Contents = InstanceProperty.normal({
+	getDefault = function()
+		return {}
+	end,
+})
```

Comment:
This property isn't exposed on the real life `LocalizationTable`, so we should avoid exposing it in Lemur I think.